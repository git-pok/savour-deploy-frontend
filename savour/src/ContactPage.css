.ContactPage-h1 {
  width: 100%;
  font-size: 60px;
  margin-bottom: 70px;
}

.ContactPage {
  display: block;
  width: 100%;
  height: 100%;
}

.ContactPage-div {
  display: inline-block;
  width: 70%;
  height: 100%;
  margin: 0px auto;
  padding: 30px 30px;
  box-sizing: content-box;
  border-radius: 10px;
  background-color: rgb(216, 216, 216);
}

.ContactPage-subtitle {
  text-align: center;
  display: block;
  font-size: 26px;
  margin: 0px auto -10px auto;
}

.ContactPage-short-text {
  display: block;
  width: 90%;
  margin: 10px auto 20px auto;
  text-align: center;
}