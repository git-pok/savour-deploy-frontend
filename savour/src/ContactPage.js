import './ContactPage.css';

/**
 * ContactPage
 * Props: none.
 * Renders the contact page.
*/
const ContactPage = () => {

  return (
    <>
      <h1 className="ContactPage-h1">App Creator and Developer Information</h1>
      <div className="ContactPage">
        <div className="ContactPage-div">
          <h2 className="ContactPage-subtitle">Name</h2>
          <p className="ContactPage-short-text">Vincent</p>
          <h2 className="ContactPage-subtitle">Nickname</h2>
          <p className="ContactPage-short-text">Fvin</p>
          <h2 className="ContactPage-subtitle">Email</h2>
          <p className="ContactPage-short-text">vincent.impreveduto.work@gmail.com</p>
          <h2 className="ContactPage-subtitle">Location</h2>
          <p className="ContactPage-short-text">Tampa Florida</p>
        </div>
      </div>
    </>
  );
}

export default ContactPage;