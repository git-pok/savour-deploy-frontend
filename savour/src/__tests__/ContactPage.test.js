import ContactPage from '../ContactPage.js';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';

test("renders ContactPage without crashing", () => {
  render(
    <MemoryRouter>
      <ContactPage />
    </MemoryRouter>
  );
});

test("renders snapshot without crashing", () => {
  const { asFragment } = render(
    <MemoryRouter>
      <ContactPage />
    </MemoryRouter>
  );

  expect(asFragment()).toMatchSnapshot();
});
