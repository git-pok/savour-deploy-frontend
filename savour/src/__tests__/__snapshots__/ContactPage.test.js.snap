// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders snapshot without crashing 1`] = `
<DocumentFragment>
  <h1
    class="ContactPage-h1"
  >
    App Creator and Developer Information
  </h1>
  <div
    class="ContactPage"
  >
    <div
      class="ContactPage-div"
    >
      <h2
        class="ContactPage-subtitle"
      >
        Name
      </h2>
      <p
        class="ContactPage-short-text"
      >
        Vincent
      </p>
      <h2
        class="ContactPage-subtitle"
      >
        Nickname
      </h2>
      <p
        class="ContactPage-short-text"
      >
        Fvin
      </p>
      <h2
        class="ContactPage-subtitle"
      >
        Email
      </h2>
      <p
        class="ContactPage-short-text"
      >
        vincent.impreveduto.work@gmail.com
      </p>
      <h2
        class="ContactPage-subtitle"
      >
        Location
      </h2>
      <p
        class="ContactPage-short-text"
      >
        Tampa Florida
      </p>
    </div>
  </div>
</DocumentFragment>
`;
